### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by joenas** [netbrick/lita-redmine#1]
```
config.handlers.redmine.url = "https://redmine.site.cz"
config.handlers.redmine.secret_key = "<secret key to encrypt tokens in redis>"
```

It should be `config.handles.redmine2`.
Also a note about `secret_key` having a max length of 56 would be helpful!

**Comment by jindrichskupa** [comment]
Hello, I dont thing so, official Lita's documentation says:
Lita.config.handlers.handler_with_config.api_key and users can set a value for it in their lita_config.rb file. The handler can then use the value the user sets by calling config.api_key in any instance method. The config instance method is a convenience accessor equivalent to robot.config.handlers.your_handler_name.

I will add note about secret_key length anywhere from 4 to 56 bytes.

**Comment by jindrichskupa** [comment]
Ok, I am sorry. My mistake - You mean name of handler - not redmine bud redmine2.

**Comment by jindrichskupa** [comment]
Fixed in commit 138374e

**Comment by joenas** [comment]
👍🏻
